Update sandbox.rst

CEKrause opened the pull request:
Adding some additional text in line with the new draft terms and conditions.

robbibt commented on setup/Sandbox/sandbox.rst:
```diff
@@ -52,11 +60,23 @@ extension. The JupyterLab interface also supports plain text and Markdown files.
-   overwritten as part of the automatic update process. To avoid this, we
-   recommend advanced users use Git to clone a new copy of ``dea-notebook``
-   into the Sandbox (`see guide here`_).
+Some things to know about The Sandbox
```
Something like "Important information about the Sandbox" might sound less informal

robbibt commented on setup/Sandbox/sandbox.rst:
```diff
@@ -52,11 +60,23 @@ extension. The JupyterLab interface also supports plain text and Markdown files.
+The Sandbox is not a production environment and should be used for protyping and exploring
+DEA's data and tools. Changes made to Jupyter notebooks in the DEA Sandbox may be automatically 
+overwritten as part of the automatic update process. To avoid this, we recommend advanced 
+users use Git to clone a new copy of ``dea-notebook`` into the Sandbox (`see guide here`_).
```
``dea-notebooks``

CEKrause replied:
Thanks.

robbibt commented on setup/Sandbox/sandbox.rst:
```diff
@@ -52,11 +60,23 @@ extension. The JupyterLab interface also supports plain text and Markdown files.
+-------------------------------
+
+The Sandbox is not a production environment and should be used for protyping and exploring
+DEA's data and tools. Changes made to Jupyter notebooks in the DEA Sandbox may be automatically 
```
This still seems more like a general "be cautious" disclaimer than a more serious warning that any changes to the default notebooks will be overwritten every time a user restarts their server. I think this is fine for now, but at some point we should try and find a good place for that more specific warning (perhaps somewhere on the Sandbox side where it's impossible to miss)

robbibt replied:
(I'm still haunted by losing an entire day of work to this back in the early days of the Sandbox, and suspect it may be a more common issue for new users than we realise)

robbibt reviewed (approved):
Looks great! Some super minor comments, feel free to ignore!

CEKrause commented:
@robbibt I have addressed your comments. Thanks! I'll merge this in once the checks pass.

I agree about making the warnings about potential lost work more prominent. I added a little bit of extra text here, but I feel like here is too buried to be useful. I'll look again at adding it to the welcome info I am writing up for The Sandbox landing page.
